plugins/content/theme/index.js

```diff
@@ -4,23 +4,23 @@
  *
  */
 
-var origin = require('../../../'),
-    contentmanager = require('../../../lib/contentmanager'),
-    rest = require('../../../lib/rest'),
-    BowerPlugin = require('../bower'),
-    ContentPlugin = contentmanager.ContentPlugin,
-    ContentTypeError = contentmanager.errors.ContentTypeError,
-    configuration = require('../../../lib/configuration'),
-    usermanager = require('../../../lib/usermanager'),
-    database = require('../../../lib/database'),
-    logger = require('../../../lib/logger'),
-    defaultOptions = require('./defaults.json'),
-    bower = require('bower'),
-    async = require('async'),
-    fs = require('fs'),
-    _ = require('underscore'),
-    util = require('util'),
-    path = require('path');
+const async = require('async');
+const bower = require('bower');
+const BowerPlugin = require('../bower');
+const configuration = require('../../../lib/configuration');
+const contentmanager = require('../../../lib/contentmanager');
+const ContentPlugin = contentmanager.ContentPlugin;
+const ContentTypeError = contentmanager.errors.ContentTypeError;
+const database = require('../../../lib/database');
+const defaultOptions = require('./defaults.json');
+const fs = require('fs');
+const logger = require('../../../lib/logger');
+const origin = require('../../../');
+const path = require('path');
+const rest = require('../../../lib/rest');
+const usermanager = require('../../../lib/usermanager');
+const util = require('util');
+const _ = require('underscore');
```

Comment:
Not sure that we need Underscore declared here?

Reply:
You're right (there were a few others as well, I've removed them)